pages/_app.js

```diff
@@ -39,7 +39,8 @@ export default function App({ Component, pageProps }) {
         <link rel="manifest" href={`${basePath}/site.webmanifest`} />
         <link rel="mask-icon" href={`${basePath}/safari-pinned-tab.svg`} color="#5bbad5" />
         <meta name="msapplication-TileColor" content="#da532c" />
-        <meta name="theme-color" content="#ffffff" />
+         <meta name="theme-color" content="##fafafa" media="(prefers-color-scheme: light)" />
+        <meta name="theme-color" content="##2f2f2f" media="(prefers-color-scheme: dark)" />
```

Comment:
Same here, redundant `#`.

Reply:
Thanks! yeah, I had both iterations with one `#` first, but apparently the header theme stopped changing between dark and light mode, so I added it back which fixed my deployment. Did the header change for you with one `#`?

Reply:
Sorry, seems like I forgot to reply yesterday - I'll take a look in a bit, because it should technically work; although no, I haven't tried out yet.